src/components/views/donate/helpers.tsx

```diff
@@ -30,12 +32,20 @@ export const prepareTokenList = (tokens: IProjectAcceptedToken[]) => {
 	return _tokens;
 };
 
-export const getNetworkNames = (networks: number[], text: string) => {
+export const getNetworkNames = (
+	networks: INetworkIdWithChain[],
+	text: string,
+) => {
 	return networks.map((network, index) => {
-		const name = config.EVM_NETWORKS_CONFIG[network]?.name;
+		// Access the network name using networkId or chainType based on the chainType
+		const name =
+			network.chainType === ChainType.EVM
+				? config.EVM_NETWORKS_CONFIG[network.networkId]?.name
+				: config.NON_EVM_NETWORKS_CONFIG[network.chainType]?.name;
+
```

Comment:
We have `getChainName` which works for EVM chains. However, I think it would be awesome to refactor it because it's not readable enough and made by assuming network id of non evm chains are always zero! which is not a strict rule and later we may modify it to non-zero and cause issues.